### PR TITLE
[TEVA-4386] new summary copy and separate feedback page

### PR DESCRIPTION
**Pull request by alexbowen**
Screenshot 2022-08-30 at 15 56 00

https://dfedigital.atlassian.net/browse/TEVA-4386

(Also little change for @adamsilver as he wants Active Jobs (19) for e.g to be all bold text)

### CODE REVIEW

File: app/controllers/publishers/vacancies/feedbacks_controller.rb [hunk 1]
```diff
@@ -1,13 +1,18 @@
 class Publishers::Vacancies::FeedbacksController < Publishers::Vacancies::BaseController
+  def new
+    @vacancy = VacancyPresenter.new(Vacancy.published.find(params[:job_id]))
```
Comment (josephhull676) [inline app/controllers/publishers/vacancies/feedbacks_controller.rb:3]: Not sure if you need `@vacancy` here. 

The vacancy is available as `vacancy`. 

Take a look at `app/controllers/publishers/vacancies/base_controller.rb`

File: app/controllers/publishers/vacancies/feedbacks_controller.rb [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+    @feedback_form = Publishers::JobListing::FeedbackForm.new
+  end
+
   def create
     @vacancy = VacancyPresenter.new(Vacancy.published.find(params[:job_id]))
```
Comment (josephhull676) [inline app/controllers/publishers/vacancies/feedbacks_controller.rb:8]: Same here, no need to set `@vacancy`

Below in `feedback_attributes` change `@vacancy` to `vacancy`

File: app/views/publishers/vacancies/feedbacks/new.html.slim [hunk 3]
```diff
@@ -0,0 +1,25 @@
+- content_for :page_title_prefix, t(".page_title", organisation: current_organisation.name)
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    h1.govuk-heading-m = t(".heading")
+    p.govuk-body = t(".description")
+
+    = form_for @feedback_form, url: organisation_job_feedback_path(@vacancy.id) do |f|
```
Comment (josephhull676) [inline app/views/publishers/vacancies/feedbacks/new.html.slim:8]: Change to `vacancy`

File: config/locales/publishers.yml [hunk 4]
```diff
@@ -319,20 +319,27 @@ en:
...
-          The job listing will appear on the service until %{application_deadline}, after which it will no longer be
-          visible to jobseekers.
-        date_posted: Your job listing will be posted on %{date}.
-        date_posted_now: Your job listing has been posted.
-        feedback:
+        date_posted: Your job listing will be published on %{date}.
+        date_posted_now: Your job listing has been published.
+        feedback_link_html: <a href="%{url}" class="govuk-link">Give feedback about this service</a> (takes 30 seconds).
+        give_feedback: Give feedback
+        heading_published: Job listing published
```
Comment (josephhull676) [inline config/locales/publishers.yml:326]: Perhaps having a `heading` key then subsequent `published` and `scheduled` keys would be better>

File: config/locales/publishers.yml [hunk 5]
```diff
@@ -319,20 +319,27 @@ en:
...
             Your feedback can help us improve the service to better support you as well as schools around England.
           heading: Have time to share some feedback?
-        page_title: The job listing for %{organisation} has been completed
-        success: The job listing has been completed
-        view_jobs: View your job listings
-        what_happens_next: What happens next
+          page_title: Feedback
+          success: The job listing has been completed
+          view_jobs: View your job listings
+          what_happens_next: What happens next
```
Comment (josephhull676) [inline config/locales/publishers.yml:342]: Don't think this is used anymore?

File: config/locales/publishers.yml [hunk 6]
```diff
@@ -319,20 +319,27 @@ en:
...
           description: >-
             Your feedback can help us improve the service to better support you as well as schools around England.
           heading: Have time to share some feedback?
-        page_title: The job listing for %{organisation} has been completed
-        success: The job listing has been completed
-        view_jobs: View your job listings
-        what_happens_next: What happens next
+          page_title: Feedback
+          success: The job listing has been completed
+          view_jobs: View your job listings
```
Comment (josephhull676) [inline config/locales/publishers.yml:341]: Don't think this is used anymore?

File: spec/system/publishers_can_publish_a_vacancy_as_a_school_spec.rb [hunk 7]
```diff
@@ -251,15 +251,15 @@
         visit organisation_job_review_path(vacancy.id)
         click_on I18n.t("buttons.submit_job_listing")
 
-        expect(page).to have_content(I18n.t("publishers.vacancies.summary.date_expires", application_deadline: format_time_to_datetime_at(vacancy.expires_at)))
+        # expect(page).to have_content(I18n.t("publishers.vacancies.summary.date_expires", application_deadline: format_time_to_datetime_at(vacancy.expires_at)))
```
Comment (josephhull676) [inline spec/system/publishers_can_publish_a_vacancy_as_a_school_spec.rb:254]: ?

File: config/locales/notifications.yml [hunk 8]
```diff
@@ -12,7 +12,7 @@ en:
       already_published: This job has already been published
       draft_saved_html: <span class="govuk-!-font-weight-bold">Draft saved</span> for %{job_title}
       feedback:
-        success: Thank you for your feedback
+        success_html: <h2 class="govuk-heading-m">Feedback sent</h2>
```
Comment (josephhull676) [inline config/locales/notifications.yml:15]: Wanted to check if this should be bold but couldn't see any designs in the ticket?

File: config/locales/publishers.yml (hunk 5 above)
Comment (alexbowen) [reply config/locales/publishers.yml]: yes its the page title tag in <head>

File: spec/system/publishers_can_publish_a_vacancy_as_a_school_spec.rb (hunk 7 above)
Comment (alexbowen) [reply spec/system/publishers_can_publish_a_vacancy_as_a_school_spec.rb]: this test isnt valid anymore as expiry info isnt displayed to the user. have removed

**Review by josephhull676 — APPROVED** [review]

**Pull request merged**